### ISSUE TITLE
Plus d'arrondissements pour Paris ?

### ISSUE THREAD

**Issue by thomas-gt** [gregoiredavid/france-geojson#24]
Hello,

Dans la dernière version (tag 2.1.0) la ville de Paris n’est plus proposée avec un découpage en arrondissements ce qui est dommage :

v2.1.0 : https://github.com/gregoiredavid/france-geojson/blob/master/departements/75-paris/communes-75-paris.geojson

v2.0.2 : https://github.com/gregoiredavid/france-geojson/blob/v2.0.2/departements/75-paris/communes-75-paris.geojson

Est-ce volontaire ? Je ne vois pas de mention spéciale à ce sujet dans le README et les commits.

Je pensais que ce type de découpage avait plus sa place dans le fichier _arrondissements-75-paris.geojson_ mais ce n’est pas le cas dans ces 2 versions (toujours sans arrondissements). 

Merci.

**Comment by gregoiredavid** [comment]
Bonjour !

Le découpage en arrondissement de Paris, Lyon et Marseille n'est pas fourni par l'IGN malheureusement. En mettant à jour le repo à partir de la nouvelle base de donnée "admin express" j'ai préféré revenir à une transcription plus fidèle plutôt que "d'injecter" manuellement ce découpage via d'autres sources comme je le faisais auparavant. L'objectif est aussi de faciliter le croisement avec d'autres bases de données qui se basent sur les communes "au sens strict".

Ceci étant dit, l'idée d'un fichier séparé me paraît bonne même si le terme "d'arrondissement" est ambigu puisqu'il correspond déjà à une entité administrative spécifique (et que le fichier [arrondissements-75-paris.geojson](https://github.com/gregoiredavid/france-geojson/blob/master/departements/75-paris/arrondissements-75-paris.geojson) existe donc déjà). Je vais tâcher de l'intégrer dans les prochaines mises à jour en trouvant le bon terme (je suis preneur d'idées bien sûr...).

**Comment by thomas-gt** [comment]
Merci pour ce retour.

La suppression des arrondissements dans les fichiers _communes-*.geojson_ ne me choque pas même si pour mon cas personnel cela m'arrangeait qu'ils soient présents (j'utilise la v2.0.2 en attendant).

Effectivement "_arrondissements_" désigne déjà les subdivision de départements. 

Apparemment la désignation exacte des subdivisions de ces 3 villes semble être "[_arrondissements municipaux_](https://fr.wikipedia.org/wiki/Arrondissement_municipal_(France))".  Une piste peut être 😃 

En tout cas merci pour ce repo très utile.

**Comment by gregoiredavid** [comment]
Oui, "arrondissements-municipaux-75-paris.geojson" par exemple me semble bien, je le note pour les prochaines mises à jour (et je garde cette issue ouverte en attendant).

Merci pour la remarque et les encouragements !
